### DISCUSSION TITLE
`TagGroup` doesn't implement `FieldError` like other field components, why?

### DISCUSSION THREAD

**Discussion by ArrayKnight** [adobe/react-spectrum#7072] [Q&A]
I was noticing that the documentation for `TagGroup` was inconsistent with other fields, where the docs show conditionally rendering a `<Text slot="errorMessage">` instead of utilizing the `FieldError` component that other fields use. Looking at the source, it appears that `TagGroup` doesn't implement the `FieldErrorContext`, which made me think it was intentional.

Can you explain the reasoning behind this inconsistency, please?

**Comment by reidbarber** [comment]
`FieldError` uses native browser validation messages, and for `TagGroup` there is no native input element to give errors for.
